cmd/soroban-rpc/lib/preflight/src/fees.rs

```diff
@@ -239,20 +232,28 @@ fn calculate_unmodified_ledger_entry_bytes(
     ledger_entries: &Vec<LedgerKey>,
     pre_storage: &ledger_storage::LedgerStorage,
     include_expired: bool,
-) -> Result<u32, Box<dyn error::Error>> {
+) -> Result<u32> {
     let mut res: usize = 0;
     for lk in ledger_entries {
-        let key_size = lk.to_xdr()?.len();
-        let entry_size = pre_storage.get_xdr(lk, include_expired)?.len();
+        let key_xdr = lk
+            .to_xdr()
+            .with_context(|| format!("cannot marshall ledger key {lk:?}"))?;
+        let key_size = key_xdr.len();
+        let entry_xdr = pre_storage
+            .get_xdr(lk, include_expired)
+            .with_context(|| format!("cannot get xdr of ledger entry with key {lk:?}"))?;
+        let entry_size = entry_xdr.len();
         res += key_size + entry_size;
     }
     Ok(res as u32)
 }
 
-fn calculate_event_size_bytes(events: &Vec<DiagnosticEvent>) -> Result<u32, Box<dyn error::Error>> {
+fn calculate_event_size_bytes(events: &Vec<DiagnosticEvent>) -> Result<u32> {
     let mut res: u32 = 0;
     for e in events {
-        let event_xdr = e.to_xdr()?;
+        let event_xdr = e
+            .to_xdr()
+            .with_context(|| format!("cannot marshall event {e:?}"))?;
```

Comment:
Do we use `marshal` or `marshall`? I can never remember which we use in this codebase...

Reply:
I think it's marshal, thanks